cypress/integration/notes.spec.js

```diff
@@ -48,5 +48,15 @@ describe("Notes", () => {
             cy.url().should("contain", "/supervision/deputies/public-authority/deputy/1/notes");
             cy.get("body > div > main > div.moj-banner.moj-banner--success > div").should("contain", "Note added");
         })
+
+        it("shows new note on the timeline", () => {
+            cy.visit("/supervision/deputies/public-authority/deputy/1/notes/add-note");
+            cy.get("#title").type("New note title")
+            cy.get("#note").type("Note text entered")
+            cy.get('form').submit()
```

Comment:
Worth being specific on which form you are submitting or this test become brittle once we start adding search forms and so on. Usually worth using action, id or a custom data attribute of the form and referencing that.